### PR TITLE
[ZEPPELIN-6140] Add logs to the runParagraph method

### PR DESCRIPTION
**Pull request by ParkGyeongTae**
### What is this PR for?
This involves revising the description of the runParagraph method and adding clear logs for each step.

### What type of PR is it?
Improvement

### Todos
* [x] - NotebookService.java

### What is the Jira issue?
* https://issues.apache.org/jira/projects/ZEPPELIN/issues/ZEPPELIN-6140

### How should this be tested?
* Debugging

### Screenshots (if appropriate)
image

### Questions:
* Does the license files need to update? No.
* Is there breaking changes for older versions? No.
* Does this needs documentation? No.

### CODE REVIEW

**Review by Reamer — COMMENTED** [review]
Only minor changes. A uniform appearance in the logs is important to me. We should avoid using “Paragraph with ID xzy” once and “Paragraph xyz” once.

File: zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java [hunk 1]
```diff
@@ -420,22 +427,25 @@ public boolean runParagraph(Note note,
...
                               ServiceCallback<Paragraph> callback) throws IOException {
 
-
     if (note == null) {
+      LOGGER.info("Failed to run paragraph: Note is null");
       return false;
     }
     LOGGER.info("Start to run paragraph: {} of note: {}", paragraphId, note.getId());
     if (!checkPermission(note.getId(), Permission.RUNNER, Message.OP.RUN_PARAGRAPH, context, callback)) {
+      LOGGER.info("Permission check failed for running paragraph: {} of note: {}", paragraphId, note.getId());
```
[the remark this answers is not in the document]
Comment (ParkGyeongTae) [reply zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java:436]: @Reamer 
I have completed this task. :)

File: zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java [hunk 2]
```diff
@@ -420,22 +427,25 @@ public boolean runParagraph(Note note,
...
       return false;
     }
 
-
     Paragraph p = note.getParagraph(paragraphId);
+
```
[the remark this answers is not in the document]
Comment (ParkGyeongTae) [reply zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java:441]: @Reamer 
I have completed this task. :)

File: zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java [hunk 3]
```diff
@@ -420,22 +427,25 @@ public boolean runParagraph(Note note,
...
     if (p == null) {
+      LOGGER.info("Paragraph with ID {} not found in note: {}", paragraphId, note.getId());
```
[the remark this answers is not in the document]
Comment (ParkGyeongTae) [reply zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java:443]: @Reamer 
I have completed this task. :)

**Review by Reamer — APPROVED** [review]
LGTM

**Comment by Reamer** [comment]
Merged into master/branch-0.12

**Pull request merged**